sdk/devcenter/Azure.Developer.DevCenter/CHANGELOG.md

```diff
@@ -1,14 +1,7 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+## 1.0.0-beta.2 (2023-02-07)
```

Comment:
Consider adding what changed between the 2 revisions.

Reply:
Added more details to the changelog.

Reply:
Thank you!